packages/media-sound/cmus/cmus.exlib

```diff
@@ -77,8 +81,8 @@ DEFAULT_SRC_CONFIGURE_PARAMS=(
 DEFAULT_SRC_COMPILE_PARAMS=( V=2 ) # Maximum verbosity
 
 SRC_CONFIGURE_OPTIONS=(
-    aac alsa ao cue ffmpeg flac jack "libsamplerate samplerate" mikmod modplug
-    "mp3 mad" mp4 "musepack mpc" opus oss "pulseaudio pulse" vorbis wav wavpack
+    aac alsa ao "elogind mpris" ffmpeg flac jack "libsamplerate samplerate" mikmod modplug
```

Comment:
elogind and system aren't options, but suboptions with a "providers" prefix in this case.